### DISCUSSION TITLE
No Dashboard anywhere

### DISCUSSION THREAD

**Discussion by addboy58** [dwainscheeren/dwains-lovelace-dashboard#415] [General]
My Baseboards is blank on my PC, iPad and Eco Show 15. How do I correct this?
thx
Dave

**Comment by ElyFNV** [comment]
Same here... After 2022.03 update

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
